server/events/apply_command_runner.go

```diff
@@ -98,16 +103,19 @@ func (a *ApplyCommandRunner) Run(ctx *CommandContext, cmd *CommentCommand) {
 	// We do this here because when we set a "Pending" status, if users have
 	// required the Atlantis status checks to pass, then we've now changed
 	// the mergeability status of the pull request.
-	ctx.PullMergeable, err = a.vcsClient.PullIsMergeable(baseRepo, pull)
+	// This sets the approved, mergeable, and sqlocked status in the context.
+	ctx.PullRequestStatus, err = a.pullReqStatusFetcher.FetchPullStatus(baseRepo, pull)
+	ctx.Log.Debug(fmt.Sprintf("Status for Repo: %v fetched: %+v", ctx.HeadRepo.FullName, ctx.PullRequestStatus))
+
 	if err != nil {
 		// On error we continue the request with mergeable assumed false.
 		// We want to continue because not all apply's will need this status,
 		// only if they rely on the mergeability requirement.
-		ctx.PullMergeable = false
-		ctx.Log.Warn("unable to get mergeable status: %s. Continuing with mergeable assumed false", err)
+		// PullRequestStatus sets approved, mergeable, and sqlocked to false when error.
+		ctx.Log.Warn("unable to get pull request status: %s. Continuing with mergeable, approved, and sqlocked assumed false", err)
 	}
 
-	ctx.Log.Info("pull request mergeable status: %t", ctx.PullMergeable)
+	ctx.Log.Info("pull request mergeable status: %t", ctx.PullRequestStatus.Mergeable)
```

Comment:
let's just remove this line, we need to start reducing the lines of noise in our logs anyways.

Reply:
Done!